package entity;

public class OrderItem {

    private int orderItemId;
    private int poId;
    private int productId;
    private int quantity;
    private double price;

    public OrderItem() {}

    public OrderItem(int orderItemId, int poId, int productId, int quantity, double price) {
        this.orderItemId = orderItemId;
        this.poId = poId;
        this.productId = productId;
        this.quantity = quantity;
        this.price = price;
    }

    public int getOrderItemId() { return orderItemId; }
    public void setOrderItemId(int orderItemId) { this.orderItemId = orderItemId; }

    public int getPoId() { return poId; }
    public void setPoId(int poId) { this.poId = poId; }

    public int getProductId() { return productId; }
    public void setProductId(int productId) { this.productId = productId; }

    public int getQuantity() { return quantity; }
    public void setQuantity(int quantity) { this.quantity = quantity; }

    public double getPrice() { return price; }
    public void setPrice(double price) { this.price = price; }
}
